modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -173,13 +173,13 @@
   "loginPageBadCredentials": "Your credentials are incorrect, or there is no such user",
   "loginPageTitle": "Login",
   "loginRequired": "Login Required",
-  "loginMaxAttemptsReached": "Too many login attempts. You may try again in a minute.",
-  "loginMaxAttemptsReached_plural": "Too many login attempts. You may try again in {{ count }} minutes.",
   "logOut": "Log Out",
   "logOutNotLoggedIn": "You were not logged in.",
   "manageDocType": "Manage {{ type }}",
   "manageDraftSubmissions": "Manage Draft Submissions",
   "managePages": "Manage Pages",
+  "maxAttemptsReached": "Too many login attempts. You may try again in a minute.",
```

Comment:
I hate to be a pain... but I meant the opposite 😄 

It is OK that the *key* contains login. That makes sense. It is one of the phrases from the module.

But the *translation* (the English text) should not because we don't want people to think they have to go all the way back to the beginning, which "login" might imply.

Reply:
ahahah Ok I get it, my bad on this one.